src/components/data-processing/GenesVsUMIs/GenesVsUMIs.jsx

```diff
@@ -58,8 +58,6 @@ const GenesVsUMIs = (props) => {
 
   useEffect(() => {
     if (!config) {
-      const newConfig = _.clone(config);
-      _.merge(newConfig, expConfig);
```

Comment:
Why do you separate `expConfig` from `plotConfig`?

Reply:
This was not doing anything so I removed it